x-pack/test/functional/apps/machine_learning/data_frame_analytics/classification_creation.ts

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import expect from '@kbn/expect';
+
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+export default function({ getService }: FtrProviderContext) {
+  const esArchiver = getService('esArchiver');
+  const ml = getService('ml');
+
+  describe('classification creation', function() {
+    this.tags(['smoke', 'melissa']);
```

Comment:
The `melissa` tag needs to be removed before merging!

Reply:
Whoops! Thanks for catching that - removed in f2846482f3ec4e2c26ddb4cbc61be423fe097dc8